mimetype_test.go

```diff
@@ -0,0 +1,11 @@
+package faker
+
+import (
+	"fmt"
```

Comment:
just need to remove this line, `fmt` is not used and because of that the build fails.

```
# github.com/jaswdr/faker [github.com/jaswdr/faker.test]
./mimetype_test.go:4:2: imported and not used: "fmt"
FAIL    github.com/jaswdr/faker [build failed]
```

Reply:
my mistake. resolved.